### PR TITLE
Show bid history

### PR DESCRIPTION
**Pull request by dangerousScript**

### CODE REVIEW

File: auction9-backend/handler.js [hunk 1]
```diff
@@ -294,3 +297,21 @@ export const postNewBid = async (event, context) => {
     });
   }
 };
+
+/* getTotalNumberOfBids - will return all auctions for current user
```
Comment (mjovanovic) [inline auction9-backend/handler.js:301]: we need better description here

Comment (dangerousScript) [reply auction9-backend/handler.js]: Fixed

File: auction9-backend/handler.js [hunk 2]
```diff
@@ -73,9 +71,9 @@ export const getActiveAuctions = async (event, context) => {
 export const postAuction = async (event, context) => {
   try {
     let reqBody = JSON.parse(event.body);
-    var startDate = new Date(reqBody.date_from).valueOf();
-    var endDate = new Date(reqBody.date_to).valueOf();
-    let status = startDate > Date.now() ? 'INACTIVE' : 'ACTIVE';
+    let startDate = new Date(reqBody.date_from).valueOf();
+    let endDate = new Date(reqBody.date_to).valueOf();
+    let status = startDate > Date.now() ? statuses.inactive : statuses.active;
```
Comment (SvetislavMarjanovic) [inline auction9-backend/handler.js:76]: On auction create, is it possible to create auction which start time has passed?

File: auction9-backend/handler.js [hunk 3]
```diff
@@ -282,9 +276,14 @@ export const postNewBid = async (event, context) => {
         // current userid hardcoded
         await mysql.query('INSERT INTO tbl_user_auction (`userID`, `auctionID`, `price`, `time`) VALUES (?, ?, ?, ?)', [2, auctionId, newBid, formattedTodayDate]);
         await mysql.query('UPDATE tbl_auction SET price=? WHERE auctionID=?', [newBid, auctionId]);
-        let resultsAuction = await mysql.query('SELECT * FROM tbl_auction WHERE auctionID=?', [auctionId]);
+        let updatedResultsAuction = await mysql.query('SELECT * FROM tbl_auction WHERE auctionID=?', [auctionId]);
+        let numberOfBids = await mysql.query('SELECT COUNT(*) FROM tbl_user_auction WHERE auctionID=?', [auctionId]);
```
Comment (SvetislavMarjanovic) [inline auction9-backend/handler.js:280]: Let's use alias for count(*), that is the name of attribute at line 282.

File: auction9-backend/handler.js (hunk 2 above)
Comment (dangerousScript) [reply auction9-backend/handler.js:76]: Dragana made this function, i just changed the way we use statuses. But it's not possible to create an auction which start time has passed, by default it's inactive. I added condition for that now

File: auction9-backend/handler.js (hunk 3 above)
Comment (dangerousScript) [reply auction9-backend/handler.js:280]: Okay, using alias 'numberOfBids' from query

**Review by mjovanovic — APPROVED** [review]

**Review by SvetislavMarjanovic — APPROVED** [review]

**Pull request merged**